README.md

```diff
@@ -1,2 +1,89 @@
-# ruby_capstone
-Create a console app that will help you to keep a record of different types of things you own: books.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Ruby Capstone
+
+> Create a console app that will help you to keep a record of different types of things you own: books.
+
+## Image for the app
+
+![](https://github.com/microverseinc/curriculum-ruby/raw/main/group-capstone/images/catalog_of_my_things.png)
+
+## Video link
+
+[video](https://www.loom.com/share/7d7e241f22d84e8d9ad1a3ec74f7c616)
+
```

Comment:
- [ ] [REQUIRED] Your project is very professional and I appreciate the work you have done. But there is one small thing you need to change. The video should contain a highlight about a `piece` of the code the student finds interesting or is `proud` of making. Please make this requested change in the new video. Good luck!